### ISSUE TITLE
Macros in Expressions

### ISSUE THREAD

**Issue by sergeymild** [schibsted/layout#64]
```
<macro name="HeaderHeight" value="{isCreatedByUser ? 80 : 112}"/>
<macro name="HeaderTop" value="{isCreatedByUser ? (50% - HeaderHeight) : 31}"/>
<UILabel
        outlet="title"
        backgroundColor="rgb(255, 222, 0)"
        textAlignment="left"
        font="{titleFont}"
        text="{title}"
        textColor="black"
        left="25"
        top="HeaderTop"/>
```
I have an error:
Unknown property `HeaderHeight` or UILabel for HeaderTop in `top` expression in UIView in Cell.

**Comment by nicklockwood** [comment]
That’s weird. I’ll try to reproduce it - thanks!

**Comment by nicklockwood** [comment]
Fixed in 0.5.5

**Comment by sergeymild** [comment]
Thanks
